### PR TITLE
bugfix/FOUR-13267

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
**Current Behavior:**

Submit does not work correctly

**Expected Behavior:**

Submit must work correctly

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13267

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:nexy
ci:deploy

### CODE REVIEW

File: resources/views/tasks/preview.blade.php [hunk 1]
```diff
@@ -339,7 +339,7 @@ class="card border-0"
           },
           submit(task) {
             if (this.isSelfService) {
-              ProcessMaker.alert(this.$t('Claim the Task to continue.'), 'warning');
+              window.parent.ProcessMaker.alert(this.$t('Claim the Task to continue.'), 'warning');
```
Comment (pmPaulis) [inline resources/views/tasks/preview.blade.php:342]: Please create a method in order to define in one place the this line and only call the method

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Comment by luNunezProcessmaker** [comment]
tested:
https://github.com/ProcessMaker/processmaker/assets/123644082/da2d4fc6-4836-4a34-81c8-a1261213d268

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-41c1b3a2d2.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-41c1b3a2d2.engk8s.processmaker.net

File: resources/views/tasks/preview.blade.php (hunk 1 above)
Comment (fagubla) [reply resources/views/tasks/preview.blade.php]: @pmPaulis , Actually ProcessMaker.alert is already a method. With this solution the message will be showed in the Processmaker and not in the frame
cc @luNunezProcessmaker

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-41c1b3a2d2.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-41c1b3a2d2.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-41c1b3a2d2.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Pull request merged**